Description: fix expression "allow to" in documentation
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-07-07
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=105735
Bug: https://rt.cpan.org/Ticket/Display.html?id=105735

--- a/README
+++ b/README
@@ -1071,7 +1071,7 @@
         $json = $json->loose([$enable])
 
     The unescaped [\x00-\x1f\x22\x2f\x5c] strings are invalid in JSON
-    strings and the module doesn't allow to "decode" to these (except for
+    strings and the module doesn't allow one to "decode" to these (except for
     \x2f). If $enable is true (or missing), then "decode" will accept these
     unescaped strings.
 
--- a/lib/JSON.pm
+++ b/lib/JSON.pm
@@ -1718,7 +1718,7 @@
     $json = $json->loose([$enable])
 
 The unescaped [\x00-\x1f\x22\x2f\x5c] strings are invalid in JSON strings
-and the module doesn't allow to C<decode> to these (except for \x2f).
+and the module doesn't allow one to C<decode> to these (except for \x2f).
 If C<$enable> is true (or missing), then C<decode>  will accept these
 unescaped strings.
 
--- a/lib/JSON/backportPP.pm
+++ b/lib/JSON/backportPP.pm
@@ -2446,7 +2446,7 @@
     $json = $json->loose([$enable])
 
 The unescaped [\x00-\x1f\x22\x2f\x5c] strings are invalid in JSON strings
-and the module doesn't allow to C<decode> to these (except for \x2f).
+and the module doesn't allow one to C<decode> to these (except for \x2f).
 If C<$enable> is true (or missing), then C<decode>  will accept these
 unescaped strings.
 
